src/System.Net.Sockets.Legacy/tests/FunctionalTests/DisconnectAsync.cs

```diff
@@ -15,6 +15,7 @@ public void OnCompleted(object sender, SocketAsyncEventArgs args)
         }
 
         [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
```

Comment:
Could you add a short comment next to the PlatformSpecific attribute indicating why it's platform-specific?  Is it because Disconnect isn't supported?